### PR TITLE
TS-136 - fix update and cancel buttons. Added both buttons to rejected and pending holidays

### PR DESCRIPTION
**Pull request by lee-cheung**
Commits: 
-  fixed update and cancel button. Added both buttons to rejected and pending holidays

Have one rejected and one pending holiday:
screenshot_1541006365

For rejected holiday, update end date to 2018-11-02 (Notice update and cancel buttons are on screen): 
screenshot_1541006381
screenshot_1541006387

Cancelling rejected holiday:
screenshot_1541006395
screenshot_1541006402

For pending holiday, updating end date to 2018-11-21 (Note update and cancel buttons are on screen):
screenshot_1541006417
screenshot_1541006420

Cancelling pending holidays:
screenshot_1541006427
screenshot_1541006431

### CODE REVIEW

File: src/components/Booking/StatusBar/index.js [hunk 1]
```diff
@@ -61,7 +61,7 @@ export default StatusBar;
 
 StatusBar.propTypes = {
   booking: PT.shape({
-    holId: PT.number,
+    eventID: PT.number,
```
Comment (d1wilko) [inline src/components/Booking/StatusBar/index.js]: super petty, but can we make `eventID` `eventId` throughout this PR? - then I'm happy to approve

**Review by d1wilko — CHANGES_REQUESTED** [review]
just one change in a few places and I can approve

File: src/components/Booking/StatusBar/index.js (hunk 1 above)
Comment (lee-cheung) [reply src/components/Booking/StatusBar/index.js]: I made the changes :)

**Review by d1wilko — APPROVED** [review]
looks good :)

**Review by William-Unosquare — APPROVED** [review]
Looks good

**Pull request merged**